materials/custom-properties.js

```diff
@@ -105,6 +105,7 @@ export default {
   borderRadiusInput: '6px',
   borderRadiusTag: '2px',
   borderRadiusForInput: '6px',
+  borderTopForOptionGroupSeparator: '1px solid #ccc',
```

Comment:
this file is custom generated, so you can't add this here as it will get overridden. Maybe just use the custom property ${colorGray} and don't bother with adding a token

Reply:
👍